content/user/agent/net/faq/DoesNotRecognizeThirdParty.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
Nevermind, I updated the text to indicate that only Nuget packages are being identified now.